includes/settings/tools.php

```diff
@@ -43,15 +45,8 @@ private function register_admin_menu( MainMenu $menu ) {
 	 * @since 1.0.0
 	 * @access private
 	 */
-	private function register_admin_menu_legacy() {
-		add_submenu_page(
-			Settings::PAGE_ID,
-			esc_html__( 'Tools', 'elementor' ),
-			esc_html__( 'Tools', 'elementor' ),
-			'manage_options',
-			self::PAGE_ID,
-			[ $this, 'display_settings_page' ]
-		);
+	private function register_admin_menu_legacy( Admin_Menu $admin_menu ) {
+		$admin_menu->register( static::PAGE_ID, new Tools_Menu_Item( $this ) );
```

Comment:
Just use it, without a dedicated method.

Reply:
Done